sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -91,6 +92,17 @@
         this.clientTelemetryEnabled = builder.isClientTelemetryEnabled();
         this.contentResponseOnWriteEnabled = builder.isContentResponseOnWriteEnabled();
         this.tracerProvider = new TracerProvider(TRACER);
+
+        List<Permission> permissionList = null;
```

Comment:
I would suggest this: 
```suggestion
        List<Permission> permissionList = new ArrayList<>();
        if (this.permissions != null && !this.permissions.isEmpty()) {
        ...
        }
```

Reply:
Thanks Kushagra. AsyncDocumentClient init() will check whether the permissionFeed is null or empty, so I think we should be safe to use either way.

Changed to following: 
      List<Permission> permissionList = new ArrayList<>();
        if (this.permissions != null ) {
        ...
        }